_posts/2016-11-21-recurse-center-week-8.md

```diff
@@ -0,0 +1,127 @@
+
+### Emotional 😊
+
+#### **Gratitude**
+
+This is pretty vague, but I've been feeling an underlying current of gratitude for RC.
+
+Since the start of the new batch, I've felt less active in the space. Coming and going as I please, coding on my own things alone. In general, taking a break from trying to do it all.
+
+I was worried that would just crush my social life, but instead it's left me feeling more connected to the space. Now the time I spend with people in really valuable and I feel more present during it.
+
+I'm super grateful to all the people who continue inviting me to activities and events during these times of change.
+
+#### **Anxious about the real world.**
+
+I've been thinking about what comes after RC. Full time job? Contracting? Travel? Self-employment?  
+I've been thinking about the ultimate place to live next. Vancouver? Toronto? Montreal? Seattle? New York?
+
+It's the paradox of choice. The more options there are, the harder it is to choose one.
+
+I really want to build inclusive spaces like RC. But how to fit that with a run of the mill software job? These are the questions!
+
+### Social 🎤
+
+#### **Super Diverse RC**
+
+On Monday, there was a "Skill Share" night at RC. I was blown away by the diversity of things RCers are skilled in. Improv comedy, juggling, rope-bondage - so many things!
+
+Coding is only a fraction of my interests, and so it only goes so far in connecting me with others. Learning about the other interests at RC makes me even more excited about the people here. It opens up a whole new dimension of discussions, which is super cool.
+
+This is especially true for things that break "coder" stereotypes. Programming is associated with being quiet and socially awkward. So seeing a bunch of programmers in a room doing improv comedy makes me so happy - stupid stereotypes 😊.
+
+#### **Shelter Tour Concert**
+
+On Thursday I went to see Madeon & Porter Robinson perform at Madison Square Garden. It's the 3rd time I've seen Madeon and he never disappoints 😍.
+
+I was in one of the boxes on the side. This meant a poor view of the stage, but also a giant section of space all to myself. I spent the whole busting out the dance moves 💃.
+
+### Intellectual ⌨️
+
+#### **One-on-one checkins**
+
+This week I switched to doing 1-on-1 checkins.
+
+Checkins are one of the few "highly encouraged" parts of RC. It's a lightweight piece of process for holding people accountable to their goals.
+
+Normal checkins involve meeting in small groups and saying what your plans are every morning.
+Experimental checkins are the same as normal checkins, but also involve everyone saying something they would like help on for that day.
+
+I found these great for encouraging collaboration and getting people invested in your work. Unfortunately, I also found them hit and miss.
+
+Sometimes everyone would be working on very different things, and asks for help would go unanswered.
```

Comment:
Consider using *requests* instead of *asks*. This might be a North American thing?